lib/matplotlib/animation.py

```diff
@@ -532,7 +532,6 @@ def grab_frame(self, **savefig_kwargs):
         buf = BytesIO()
         self.fig.savefig(
             buf, **{**savefig_kwargs, "format": "rgba", "dpi": self.dpi})
-        renderer = self.fig.canvas.get_renderer()
```

Comment:
This one makes me nervous as calling `get_renderer()` may force a re-cahing of the renderer if the figure size or DPI has changed.

Reply:
:sheep: never mind, reading where this is more carefully, if the figure has changed between the `savefig` directly above and here we have much bigger issues!